dist/tester/timescale_timescaledb.repo

```diff
@@ -1,5 +1,6 @@
 [timescale_timescaledb]
 name=timescale_timescaledb
+baseurl=https://packagecloud.io/timescale/timescaledb/el/7/$basearch
```

Comment:
why was this change needed.

Reply:
This was previously dynamically added to the file in the docker build:

```
echo "baseurl=https://packagecloud.io/timescale/timescaledb/el/${OS_VERSION}/\$basearch" >> /etc/yum.repos.d/timescale_timescaledb.repo
```

The reason behind this was to support both centos 7 and 8 builds. We're no longer building centos 8, and we need to find another solution for rhel 8 anyway, so I would prefer to keep this not dynamic for now.